payloads/external/SeaBIOS/Makefile.inc

```diff
@@ -70,7 +70,7 @@ ifeq ($(CONFIG_SEABIOS_VGA_COREBOOT),y)
 	echo "CONFIG_BUILD_VGABIOS=y" >> seabios/.config
 endif
 
-	echo "CONFIG_DEBUG_LEVEL=0" >> seabios/.config
+	echo "CONFIG_DEBUG_LEVEL=1" >> seabios/.config
```

Comment:
CONFIG_DEBUG_LEVEL has to be at least 1, even when using dprintf level 0 or output is disabled.